packages/react-cosmos/src/config/shared.ts

```diff
@@ -12,6 +12,13 @@ export type CosmosConfig = {
   // unspecified IPv4 address (0.0.0.0) otherwise.
   hostname: null | string;
   port: number;
+  https?: boolean;
+  httpsOptions?: {
+    keyPath?: string;
+    certPath?: string;
+  };
```

Comment:
This is probably a pet peeve, but I prefer having all options _defined_ (no `?:`) in `CosmosConfig`. This means that `https` is always a boolean that defaults to `false`, `httpsOptions` can either be `null` or have a value, in which case `keyPath` and `certPath` will always a string. You can achieve this type guarantees by providing defaults in `createCosmosConfig.ts` option helpers, like the other options are handled.